NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Breaking changes
 
+* `path_expand()` and `path_home()` now use `USERPROFILE` or
+  `HOMEDRIVE`/`HOMEPATH` as the user home directory on Windows. This differs
+  from the definition used in `path.expand()` but is consistent with
+  definitions from other programming environments such as python and rust. To
+  mimic R's (and previous) behavior use `Sys.setenv("R_FS_HOME" = Sys.getenv("R_USER"))`
+
```

Comment:
You could also point out that this is also more compatible with Git (and ssh), both of which, by default, put their user-level files in `USERPROFILE`.

Either here or in the help.

Reply:
Summary of my research on this re: ssh keys:

https://github.com/ropensci/git2r/issues/314#issuecomment-359135124

Reply:
Or both here and the help!